src/Core/MySQL/MySQLReplication.h

```diff
@@ -292,6 +292,24 @@ namespace MySQLReplication
         MYSQL_DELETE_ROWS_EVENT = 4,
     };
 
+    inline String to_string(MySQLEventType type)
```

Comment:
`to_string -> toString` is better with our code style

Reply:
> `to_string -> toString` is better with our code style

https://github.com/ClickHouse/ClickHouse/pull/52204/files/89073f4a0ac18d503e57979e14cf5f425eae6c1f#diff-a9b857de34fc85265ad90e053ece16aa4e1ada59ed7775c0eb5783968d96eec2R190

Reply:
hm, ok, let's leave it as it is now.